react/Alerter/index.jsx

```diff
@@ -18,7 +18,7 @@ class Alerter extends Component {
 
   componentDidMount () {
     if (this.shouldAutoClose) {
-      const closeDelay = this.props.duration ? parseInt(this.props.duration) : DEFAULT_AUTOCLOSE_DELAY;
+      const closeDelay = this.props.duration ? parseInt(this.props.duration) : DEFAULT_AUTOCLOSE_DELAY
```

Comment:
Prefer `parseInt(this.props.duration, 10)` to be sure we parse the duration in base 10 ;)